web/middleware/requestlogging/requestlogging.go

```diff
@@ -79,6 +83,11 @@ func (l *zerologLogEntry) Write(status, bytes int, header http.Header, elapsed t
 	}
 }
 
+func (l *zerologLogEntry) pkgName() string {
+	pkgPathParts := strings.Split(reflect.TypeOf(*l).PkgPath(), "/")
```

Comment:
reflect is a bad idea in production code, but I'll set the field log.logger and reduce WARN to INFO for better compatibility.